### PR TITLE
add instructions for ami

### PR DESCRIPTION
**Pull request by lightbiaggi**
This pr contains the readme instructions to indicate what the ami contains and how to use them .
Note: i assumed the user has knowledge how to provision an EC2 we can add link to that instruction

### CODE REVIEW

File: docs/src/install.md [hunk 1]
```diff
@@ -6,6 +6,32 @@ to install the latest stable version of this package:
 ```julia
 pkg> add Bloqade
 ```
+## Using Bloqade Optimized AMIs from AWS Marketplace
+The Bloqade team has implemented 2 dedicated AMIs that can be acquired using AWS Marketplace
+
+- Bloqade AMI with Julia
+- Bloqade Optimized Deep Learning AMI with CUDA and Julia
```
Comment (Roger-luo) [inline docs/src/install.md:13]: Maybe we can just say CUDA? There is nothing to do with deep learning for us IIUC

File: docs/src/install.md [hunk 2]
```diff
@@ -6,6 +6,32 @@ to install the latest stable version of this package:
...
+
+
+### Bloqade AMI with Julia (base image)
+The Bloqade AMI contains:
+- The latest Julia installation using juliaup '1.5.1' which Julia '1.5.1'  for more information please refer to [Juliaup](https://github.com/JuliaLang/juliaup)
```
Comment (Roger-luo) [inline docs/src/install.md:18]: we should do 

```sh
juliaup add release
```

as default, since we always want the latest stable release.

Comment (Roger-luo) [reply docs/src/install.md:18]:
```suggestion
- The latest Julia installation using `juliaup add release` which setup the latest Julia release. For more information please refer to [Juliaup](https://github.com/JuliaLang/juliaup) 
```

File: docs/src/install.md [hunk 3]
```diff
@@ -6,6 +6,32 @@ to install the latest stable version of this package:
...
+
+### Bloqade AMI with Julia (base image)
+The Bloqade AMI contains:
+- The latest Julia installation using juliaup '1.5.1' which Julia '1.5.1'  for more information please refer to [Juliaup](https://github.com/JuliaLang/juliaup) 
+- The latest version of Bloqade 
+
+### Bloqade Optimized Deep Learning AMI with CUDA and Julia
+In addition to the content of our base image, this image further contains
+- NVIDIA CUDA, cuDNN, NCCL, GPU Drivers, Intel MKL-DNN, Docker, NVIDIA-Docker and EFA support
+- Block devices
```
Comment (Roger-luo) [inline docs/src/install.md:24]: I'm wondering if there is a better way to present this? Maybe put them as a table or at least put the paths into a code block?

File: docs/src/install.md [hunk 4]
```diff
@@ -6,6 +6,32 @@ to install the latest stable version of this package:
...
+ /dev/sda1=snap-03d72fbeb983a4663:60:true:gp2
+ /dev/sdb=ephemeral0
+ /dev/sdc=ephemeral1
+
+
+### Bloqade AMI
+To use the AMI
```
Comment (Roger-luo) [inline docs/src/install.md:31]: I think a warning block would be better to notify the users that the AMI is not public ready yet?

File: docs/src/install.md [hunk 5]
```diff
@@ -16,7 +42,7 @@ results in a binary to accelerate its loading/compilation
 time. This is useful when you have lots of interactive
 programming needs with Bloqade.
 
-To build system image for your environment, please use
+To build a system image for your environment, please use
```
Comment (Roger-luo) [inline docs/src/install.md:45]: good catch!

File: docs/src/install.md [hunk 6]
```diff
@@ -61,12 +89,11 @@ cd Bloqade
 .ci/run dev
 ```
 
-How this works? `.ci/run dev` command actually calls the `Pkg.develop`
+How do this work? The `.ci/run dev` command actually calls the `Pkg.develop`
```
Comment (Roger-luo) [inline docs/src/install.md:92]:
```suggestion
How does this work? The `.ci/run dev` command actually calls the `Pkg.develop`
```

**Review by Roger-luo — CHANGES_REQUESTED** [review]
I think this is in general looks good, perhaps @Wang-Shengtao or @fanglifl  can take a second look as someone less familiar with AWS services.

File: docs/src/install.md (hunk 1 above)
Comment (lightbiaggi) [reply docs/src/install.md:13]: yes sir

File: docs/src/install.md (hunk 3 above)
Comment (lightbiaggi) [reply docs/src/install.md:24]: i was thinking a table but was worried overkill? hmm

File: docs/src/install.md (hunk 4 above)
Comment (lightbiaggi) [reply docs/src/install.md:31]: yes agreed let me tweek it that was my concern we change it once it is official

**Comment by lightbiaggi** [comment]
@Roger-luo  @Wang-Shengtao  @plquera  Updated pr based on team's suggestion let me know

**Pull request merged**